bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -618,7 +618,7 @@ private function getDataForField($object, $key, $fielddefinition, $objectFromVer
 
             $relations = $object->getRelationData($refKey, !$fielddefinition instanceof ReverseObjectRelation, $refId);
 
-            if (empty($relations) && !empty($parent)) {
+            if (empty($relations) && !empty($parent) && $fielddefinition->supportsInheritance()) {
```

Comment:
I would check if field supports inheritance before checking relations & parent
```suggestion
            if ($fielddefinition->supportsInheritance() && empty($relations) && !empty($parent)) {
```